docs/LSPSupport.md

```diff
@@ -125,7 +125,8 @@ Here is an example with the [Qute language server](https://github.com/redhat-dev
 #### Hover
 
 [textDocument/hover](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_hover) is implemented with `documentationProvider` extension point to support any language.
-However as IJ `documentationProvider` doesn't support aggregation of several documentationProvider, it takes
+
+However, as IJ `documentationProvider` doesn't support aggregation of several documentationProvider, it takes
 the first `documentationProvider` which found.

```

Comment:
the first `documentationProvider` that is found.

Reply:
fixed